src/lexer.js

```diff
@@ -90,16 +90,30 @@ function copyLE (src, outBuf16) {
     outBuf16[i] = src.charCodeAt(i++);
 }
 
+function getWasmBytes() {
+  const binary = 'WASM_BINARY';  // This string will be replaced by build.js.
+  if (typeof window !== 'undefined' && typeof atob === 'function')
+    return Uint8Array.from(atob(binary), x => x.charCodeAt(0));
```

Comment:
This comes from a formatting of the original code but I am actually not sure why there is this `x => x.charCodeAt(0)`, `Uint8Array.from` only takes one argument AFAIK.

Reply:
It takes three arguments, and the second one is to map from whatever the first argument is made of to numbers: https://tc39.es/ecma262/#sec-%typedarray%.from

Reply:
Ah I see. Thanks for the clarification, I was probably looking at the wrong docs.